EnglishFilter/sections/foreign.txt

```diff
@@ -4827,6 +4827,10 @@ fxp.co.il#@#iframe[width="300"][height="250"]
 !------------------------------ Italian ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80436
+gdr-online.com#%#//scriptlet("set-constant", "isAdsLoaded", "true")
+!+ PLATFORM(ios, ext_android_cb, ext_safari)
+@@||gdr-online.com^$generichide
```

Comment:
I guess it should be `@@||gdr-online.com/adv/ads.js` instead of `generichide` rule.

Reply:
Ah yes, I added the wrong rule. How silly of me.